Dcumentation/PxPartySquad/examples_optimization/example_PxPartySquad.sol

```diff
@@ -0,0 +1,121 @@
+function claimTreasure(uint256 _weekNumber, bytes calldata _signature) external noContracts nonReentrant {

```

Comment:
Pls add description and details of why this change is being done and  add link to work item.

Reply:
We are not changing any existing function. This is an example file from auditor.